medallion/backends/mongodb_backend.py

```diff
@@ -127,6 +130,30 @@ def _get_object_manifest(self, api_root, collection_id, filter_args, allowed_fil
             headers = get_custom_headers(manifest_resource)
             return manifest_resource, headers
 
+    def object_manifest_check(self):
+        db = self.client
+        objects_exists = False
+        for api_root in db.list_database_names():
+            cols = db[api_root].list_collection_names()
+            if "objects" in cols:
```

Comment:
Could cut down on indentation by replacing this with 
```suggestion
            if "objects" not in cols:
                continue
```